corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -378,6 +398,27 @@ def _with_progress(self, doc_types, iterable, progress_name='Migrating'):
             self.log_info("{} ({})".format(doc_count, ', '.join(doc_types)))
             return iterable
 
+    def _log_objects_processed_count(self, tags, throttled=False):
+        if throttled and self.processed_objects % 100 != 0:
+            return
+
+        datadog_counter("commcare.couchsqlmigration.docs_processed",
+                        value=self.processed_objects,
```

Comment:
nit: would be nice to use a consistent name for the local state variable and the datadog metric. My preference would be to rename `self.processed_objects` to `self.docs_processed`